app/src/main/java/comp3350/mealbuddy/objects/Edible.java

```diff
@@ -0,0 +1,24 @@
+package comp3350.mealbuddy.objects;
+
+
+import java.util.List;
+import java.util.Map;
+
+import comp3350.mealbuddy.objects.MacroFactory.Macros;
+import comp3350.mealbuddy.objects.MicroFactory.Micros;
+
+public abstract class Edible {
+
+    public String name;
+    public List<String> labels;
+    public Map<Micros, Integer> micros;
+    public Map<Macros, Integer> macros;
```

Comment:
the idea here is good, but I just don't understand the point of making these enum maps. we should discuss the implementation tmr